### PR TITLE
Move to new asset criticality levels

### PR DESCRIPTION
**Pull request by hop-dev**
Move from old to new criticality levels

- very_important -> extreme_impact
- important -> high_impact
- normal -> medium_impact
- not_important -> low_impact

### CODE REVIEW

File: createEvents.mjs [hunk 1]
```diff
@@ -4,7 +4,7 @@ import moment from 'moment';
 export default function createEvents(override = {}) {
    return {
     "@timestamp": moment().format("yyyy-MM-DDTHH:mm:ss.SSSSSSZ"),
-    criticality: faker.helpers.arrayElement(["NOT_IMPORTANT", "NORMAL", "IMPORTANT", "VERY_IMPORTANT",]),
```
Comment (hop-dev) [inline createEvents.mjs:7]: I don't think these should have been capitalised unless I am misunderstanding?

File: commands/entity-store.mjs [hunk 2]
```diff
@@ -12,11 +12,7 @@ let EVENT_INDEX_NAME = "auditbeat-8.12.0-2024.01.18-000001";
 const offset = () => Math.random() * 1000;
 
 const ASSET_CRITICALITY = [
-  "very_important",
-  "not_important",
-  "normal",
-  "important",
-  "unknown",
```
Comment (hop-dev) [inline commands/entity-store.mjs:19]: Is unknown a value that is used by the data generator? It isnt a valid criticality level

Comment (hop-dev) [reply commands/entity-store.mjs:19]: Ah yes I see https://github.com/hop-dev/security-documents-generator/blob/30dada4311d0238baf1ee937daf52cfcec5c56f6/commands/entity-store.mjs#L101

Comment (tiansivive) [reply commands/entity-store.mjs:19]: I think at one point we did have an unknown, probably why that's still here.
We can probably clean this up in the future

File: createEvents.mjs (hunk 1 above)
Comment (tiansivive) [reply createEvents.mjs:7]: I dont know. Nikita was the one who did the criticality assignment here. Maybe at one point they were capitalised and we just never changed them here? 
It's weird that it still works? Although I do remember seeing some issues where we'd see less assigned criticalities than expected so maybe this is the cause?

But yeah, should be fixed now with your changes

**Review by tiansivive — APPROVED** [review]
LGTM

**Pull request merged**